DDDEastAnglia/Areas/Admin/Views/Sponsor/Create.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿@model DDDEastAnglia.Areas.Admin.Models.SponsorModel
+
+<h2>Add Sponsor</h2>
+
+@using (Html.BeginForm("Create", "Sponsor", FormMethod.Post, new { @class = "form-horizontal", enctype = "multipart/form-data" }))
+{
+    @Html.AntiForgeryToken()
+    @Html.ValidationSummary(true)
+
+    <div class="control-group">
+        @Html.LabelFor(m => Model.Name, new { @class = "control-label" })
+        <div class="controls">
+            @Html.TextBoxFor(m => Model.Name, new { autofocus = true })
+            @Html.ValidationMessageFor(m => Model.Name)
+        </div>
+    </div>
+
+    <div class="control-group">
+        @Html.LabelFor(m => Model.Url, new { @class = "control-label" })
+        <div class="controls">
+            @Html.TextBoxFor(m => Model.Url, new { type = "url" })
+            @Html.ValidationMessageFor(m => Model.Url)
+        </div>
+    </div>
+
+    <div class="control-group">
+        @Html.LabelFor(m => Model.SponsorshipAmount, new { @class = "control-label" })
+        <div class="controls">
+            <div class="input-prepend">
+                <span class="add-on">£</span>
+                @Html.TextBoxFor(m => Model.SponsorshipAmount)
+            </div>
+            @Html.ValidationMessageFor(m => Model.SponsorshipAmount)
+        </div>
+    </div>
+
+    <div class="control-group">
+        @Html.LabelFor(m => Model.PaymentDate, new { @class = "control-label" })
+        <div class="controls">
+            @Html.TextBoxFor(m => Model.PaymentDate, new { type = "date" })
+            @Html.ValidationMessageFor(m => Model.PaymentDate)
+        </div>
+    </div>
+
+    <div class="control-group">
+        @Html.Label("Logo", new { @class = "control-label" })
+        <div class="controls">
+            <input type="file" name="Logo" data-val="true" data-val-required="A logo must be supplied." />
```

Comment:
The `required` attribute would be a useful addition here too: it will cause the browser to prompt the user to include a logo.